pkg/network/node/networkpolicy.go

```diff
@@ -624,6 +698,10 @@ func (np *networkPolicyPlugin) handleDeleteNetworkPolicy(obj interface{}) {
 
 	if npns, exists := np.namespaces[vnid]; exists {
 		np.cleanupNetworkPolicy(policy)
+		for i := range npns.policies[policy.UID].ipBlocks {
```

Comment:
I would expect this to be part of cleanupNetworkPolicy.
cleanupNetworkPolicy is a private function and it's only used here, so changing the signature of the function shouldn't be a problem.